src/main/cliapp/src/components/DiseaseAnnotationsComponent.js

```diff
@@ -163,6 +165,7 @@ export const DiseaseAnnotationsComponent = () => {
             itemTemplate={subjectItemTemplate}
             completeMethod={searchSubject}
             onChange={(e) => onSubjectEditorValueChange(props, e)}
+            forceSelection={true}
```

Comment:
Setting forceSelection as a property leads to a bug where if you tab away from an input, it causes the cell to go blank.

Reply:
Ohh yes, I accidently added that while fixing other issue. Only the .replace lines (2 times) is the change.